libfs/file.go

```diff
@@ -114,3 +118,65 @@ func (f *File) Close() error {
 	f.node = nil
 	return nil
 }
+
+func (f *File) getLockID() keybase1.LockID {
+	// We must not change or remove this since it'd break locking guarantees.
```

Comment:
Sounds good. Although since we don't have a way to *require* client to upgrade with current scheme, I'm not sure that'll work well. Perhaps we should include a way for `mdserver` to require minimum client version so that we can reject old clients once the "transition" is done?

Reply:
Yeah, we can always make the mdserver enforce a min client version.  But that's something we can add later.  If you want to be more clear about it, you could use the first byte of a lock ID as a lock ID version number.

Reply:
Oh interesting! That's a great idea. Will do.

Reply:
Made the change [here](https://github.com/keybase/client/pull/8630); will vendor it in once it's merged.